### PR TITLE
[DO NOT MERGE] testing tensorflow ci

### PR DESCRIPTION
**Pull request by marcrasi**

### CODE REVIEW

**Comment by marcrasi** [comment]
@swift-ci test tensorflow

**Comment by marcrasi** [comment]
@swift-ci test tensorflow

File: test/TensorFlow/sema_errors.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ import TensorFlow
 
 // Make sure we cleanly diagnose this.
 public func testAmbiguous() {
-  #tfop("foo")  // expected-error {{type of expression is ambiguous without more context}}
+  #tfop("foo")  // expected-error {{this is not the error I expect}}
```
Comment (marcrasi) [inline test/TensorFlow/sema_errors.swift]: test comment

**Comment by marcrasi** [comment]
@swift-ci test tensorflow

**Comment by rxwei** [comment]
Should this be closed?

**Comment by marcrasi** [comment]
Yes

**Pull request closed without merging**